src/metabase/driver/druid/query_processor.clj

```diff
@@ -530,6 +548,20 @@
         {:query      druid-query
          :query-type query-type}))))
 
+
+(defn- columns->getter-fns
+  "Given a sequence of COLUMNS keywords, return a sequence of appropriate getter functions to get values from a single result row. Normally,
+   these are just the keyword column names themselves, but for `:timestamp___int`, we'll also parse the result as an integer (for further
+   explanation, see the docstring for `units-that-need-post-processing-int-parsing`)."
+  [columns]
+  (vec (for [k columns]
+         (if (not= k :timestamp___int)
+           k
+           (comp (fn [^String s]
+                   (when (seq s)
```

Comment:
This is what's called "nil-punning" in Lisp languages. `seq` will return a (lazy) sequence of objects in a collection if the collection is non-empty, otherwise nil. For the purposes of `seq` a string is considered to be a sequence of chars so

```clojure
(seq nil)    -> nil
(seq "")     -> nil
(seq "ab") -> [\a \b]
```